README.md

```diff
@@ -28,7 +28,12 @@ Optionnally you can:
 * change the other plugin settings as required,
 * tweak the execution times of the scheduled task running the file conversions.
 
-## Issues
+Hints:
+
+* Read https://docs.moodle.org/310/en/Managing_filters how to activate and order filters.
```

Comment:
@Danie could you please change this URL to https://docs.moodle.org/en/Managing_filters to be version agnostic?

Reply:
done